tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "tsconfigs/nodejs-module",
+  "extends": "./node_modules/tsconfigs/nodejs-module.json",
```

Comment:
Just curious, what does this change? did the old stop working in the new ts version?

Reply:
Yes. `tsc` would still run fine but `npm run test` would fail with:

```
TSError: ⨯ Unable to compile TypeScript:
error TS6053: File 'tsconfigs/nodejs-module' not found.
```

A bit weird indeed.